devops/Makefile

```diff
@@ -433,6 +449,32 @@ deploy-queue-consumers:
 		| oc -n $(NAMESPACE) apply -f -
 	$(call rollout_and_wait,dc/$(QUEUE_CONSUMERS))
 
+deploy-load-test-gateway:
```

Comment:
Just for discussion, if we use this container only for load-testing, can this be used for db-migrations too? If we properly spin this up and adding the db-migrations in the Dockerfile?

Reply:
I believe there is no intention to use this container for db migrations.

Reply:
This container may be deployed only to DEV, it may be not present always, and I am struggling to see a benefit in doing it. Can you please provide a benefit for doing it?

Reply:
This was a temporary container used for load-test, so I thought instead of having 2 jobs, if we extend it for db-migrations, we can use it during revert. I am aware that this may be only deployed in DEV, in case if we extend its functionalities as a common container, I was suggesting.